regression/pages/lms/checkout_page.py

```diff
@@ -0,0 +1,58 @@
+"""
+Payment page.
+"""
+from bok_choy.page_object import PageObject
+
+
+class PaymentPage(PageObject):
+    """
+    Payment page
+    """
+    url = 'https://testsecureacceptance.cybersource.com/checkout'
+
+    def is_browser_on_page(self):
+        return 'checkout' in self.browser.title.lower()
+
+    def make_test_payment(self):
+        """
+        Fill payment on payment page with testing payment data
+        """
+        # Fill payment details
+        fill_details = {
+            '#bill_to_forename': 'first_name',
+            '#bill_to_surname': 'last_name',
+            '#bill_to_address_line1': 'address1',
+            '#bill_to_address_line2': 'address2',
+            '#bill_to_address_city': 'city',
+            '#bill_to_address_postal_code': '23as23',
+            '#bill_to_email': 'a@a.com',
+            '#card_number': '4111111111111111',
+            '#card_cvn': '937'
+        }
+        for css, value in fill_details.iteritems():
+            self.q(css=css).fill(value)
+
+        drop_downs = {
+            'bill_to_address_country': 'BH',
+            'card_expiry_month': '08',
+            'card_expiry_year': '2020'
+        }
+
+        # Select drop downs
+        for css, value in drop_downs.iteritems():
+            self.q(
+                css='select[name="{}"] option[value="{}"]'.format(css, value)
+            ).click()
+
+            self.wait_for(
+                lambda: self.q(
+                    css='select[name="{}"] option[value="{}"]'.format(
+                        css, value)
+                ).selected, "Correct value is selected"
+            )
+
+        # Click Visa type on Card Type radio button
+        self.q(css='#card_type_001').click()
```

Comment:
Should you go by the text value instead of the css location? I'm just trying to anticipate if a time comes wherein something else is card_type_001 but visa is still an option.